.rubocop.yml

```diff
@@ -387,9 +387,6 @@ Performance/StringInclude:
 Minitest/AssertNil:
   Enabled: true
 
-Minitest/AssertPredicate:
-  Enabled: true
```

Comment:
The reason for this rule is that it gives you much better error message in case of assertion failures, making debugging much easier:

e.g. 

```
Failure:
ContinuousIntegrationTest#test_report_with_only_successful_steps_combined_gives_success [test/continuous_integration_test.rb:30]:
Expected true to be nil or false
```

vs

```
Failure:
ContinuousIntegrationTest#test_report_with_only_successful_steps_combined_gives_success [test/continuous_integration_test.rb:30]:
Expected #<ActiveSupport::ContinuousIntegration:0x000000011e281b20 @results=[true, true]> to not be success?.
```

Reply:
I don't think that carries the weight for how ugly that method is. To fix what's wrong, you still have to look at the line, and when you do, you'll see what the predicate is. Appreciate that the output is marginally better, but to me that doesn't outweigh the aesthetic regression.